openedx/core/djangoapps/ace_common/template_context.py

```diff
@@ -38,6 +38,15 @@ def get_base_template_context(site):
     except NoReverseMatch:
         dashboard_url = reverse('home')
 
+    contact_mailing_address = configuration_helpers.get_value('CONTACT_MAILING_ADDRESS1')
```

Comment:
Is it typo `CONTACT_MAILING_ADDRESS1` or there is a specific reason to add the 1 ?

Reply:
typo.. already removed.

Reply:
It's fixed in latest commit.